app/src/org/commcare/views/widgets/GeoPointWidget.java

```diff
@@ -213,9 +197,9 @@ public void setBinaryData(Object answer) {
         String[] sa = s.split(" ");
         mAnswerDisplay.setText(
                 StringUtils.getStringSpannableRobust(getContext(), R.string.latitude) +
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/cb307f0c79a9e28d4bce7b60e4ee756470e4a32e/app/src/org/commcare/views/widgets/GeoPointWidget.java#L199"></span>Do not concatenate text displayed with <code>setText</code>. Use resource string with placeholders.</td>
    </tr>
  </tbody>
</table>